scripts/zones/AlTaieu/mobs/Jailer_of_Prudence.lua

```diff
@@ -19,6 +24,17 @@ end;
 -----------------------------------
 
 function onMobSpawn(mob)
+    mob:AnimationSub(0); -- Mouth closed
+    mob:addStatusEffect(EFFECT_FLEE,100,0,30);
```

Comment:
That's the one. It gains the effect, but I believe it doesn't give a message about it wearing.